### PR TITLE
Cleanup integration test setup

### PR DESCRIPTION
**Pull request by rasmus**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:6.8.5
+    container_name: elasticsearch-ef
+    environment:
+      - discovery.type=single-node
+      - ES_JAVA_OPTS=-Xms512m -Xmx512m
+    ports:
+      - "9200:9200"
+    mem_limit: 1g
```
Comment (ProH4Ck) [inline docker-compose.yml]: I tried various configuration but the only one working is 4g with `ES_JAVA_OPTS=-Xms1g -Xmx1g`

**Comment by rasmus** [comment]
Thanks, I'll revert it

**Comment by rasmus** [comment]
darn, works on my machine ™️

**Comment by rasmus** [comment]
Too old

**Pull request closed without merging**